src/libs/actions/IOU.js

```diff
@@ -101,7 +101,7 @@ function resetMoneyRequestInfo(id = '') {
         currency: lodashGet(currentUserPersonalDetails, 'localCurrencyCode', CONST.CURRENCY.USD),
         comment: '',
         participants: [],
-        merchant: CONST.TRANSACTION.DEFAULT_MERCHANT,
+        merchant: '',
```

Comment:
this will not set the merchant as `Request` default for IOU requests, right?

Reply:
Yes, as I understand from the ticket we need to make it clear, not?

Reply:
For IOUs it stays `Request` though still I think?

Reply:
We agreed on (none), no? in the conversation

Reply:
here is an outdated code, I changed reset to (none)